__tests__/app.test.js

```diff
@@ -47,4 +47,37 @@ describe("GET /api", () => {
     });
 });
 
-
+describe("/api/articles/:article_id", () => {
+    it("GET 200: returns a single article object when given a valid and existing article id", () => {
+        return request(app)
+        .get("/api/articles/5")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article.article_id).toBe(5) 
+            expect(typeof body.article.article_id).toBe("number")
```

Comment:
This is redundant because you have already checked that it is 5, so that must be a number